source/extensions/common/wasm/null/null_vm.h

```diff
@@ -0,0 +1,72 @@
+#pragma once
+
+#include <memory>
+#include <utility>
+#include <vector>
+
+#include "envoy/registry/registry.h"
+
+#include "common/common/assert.h"
+
+#include "extensions/common/wasm/null/null_vm_plugin.h"
+#include "extensions/common/wasm/well_known_names.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Common {
+namespace Wasm {
+namespace Null {
+
+struct NullVm : public WasmVm {
+  NullVm() = default;
```

Comment:
nit: del

Reply:
We need this constructor.  Do you mean remove the "= default" and replace with {} ?

Reply:
Sorry nevermind I see the copy constructor below.